src/careamics/lightning/callbacks/data_stats_callback.py

```diff
@@ -7,21 +7,21 @@
 class DataStatsCallback(Callback):
     """Callback to update model's data statistics from datamodule.
 
-    This callback ensures that the model has access to the data statistics (mean and
-    std) calculated by the datamodule before training starts.
+    This callback ensures that the model has access to the data statistics (mean, std)
+    calculated by the datamodule before training starts.
     """
 
     def setup(self, trainer: L.Trainer, module: L.LightningModule, stage: str) -> None:
         """Called when trainer is setting up.
 
         Parameters
         ----------
-        trainer : Lightning.Trainer
-            The trainer instance.
-        module : Lightning.LightningModule
-            The model being trained.
+        trainer : L.Trainer
+            PyTorch Lightning trainer.
+        module : L.LightningModule
+            Lightning module.
```

Comment:
The decision a while back was to write out the library it comes from. Although `Lightning` is not the canonical library, it is much more informative than `L`.

```suggestion
        trainer : Lightning.Trainer
            PyTorch Lightning trainer.
        module : Lightning.LightningModule
            Lightning module.
```